R/render_validation_report.R

```diff
@@ -45,18 +50,16 @@ render_validation_report <- function(report_path, output_dir = dirname(report_pa
 #' @examples
 #' \dontrun{
 #' vt_path()
-#' \dontrun{
 #' vt_path("some", "reqs", "req01.md")
 #' vt_path("some/reqs/req01.md")
 #' }
-#' }
+#'
 #'
 vt_path <- function(..., pkg = "."){
 
   state <- Sys.getenv("vt_validation_state")
```

Comment:
can we get some documentation around these environment vars? nothing fancy, just a roxygen block that lists all names?

Reply:
sure, this is mainly for passing information and state across to render